minard/RSTools.py

```diff
@@ -0,0 +1,484 @@
+# -*- coding: utf-8 -*-
+# Need comment above to use bullet-point character
+from wtforms import Form, BooleanField, StringField, PasswordField, validators
+from .db import engine, engine_nl
+from collections import OrderedDict
+import psycopg2
+import psycopg2.extras
+from datetime import datetime
+from dateutil import tz, parser
+import pytz
+from .views import app
+
+def file_list_form_builder(formobj, runlists, data):
+    class FileListForm(Form):
+        pass
+
+    for (i, listname) in enumerate(runlists.keys()):
+        listnum = runlists[listname]
+        if listnum in data and formobj == -1:
+            setattr(FileListForm, '%s' % listname, BooleanField(label=listname, default='checked'))
+        else:
+            setattr(FileListForm, '%s' % listname, BooleanField(label=listname))
+
+        setattr(FileListForm, 'name', StringField('Name', [validators.Length(min=1), validators.InputRequired(), validators.Regexp('[A-Za-z0-9\s]{1,}', message='First and second name required.')]))
+        setattr(FileListForm, 'comment', StringField('Comment', [validators.InputRequired()]))
+        setattr(FileListForm, 'password', PasswordField('Password', [validators.InputRequired()]))
+
+    if formobj != -1:
+        return FileListForm(formobj)
+    else:
+        return FileListForm()
+
+def get_current_lists_run(run):
+    conn = engine.connect()
+    result = conn.execute("SELECT list FROM evaluated_runs WHERE run={}".format(run))
+    return [int(row[0]) for row in result.fetchall()]
+
+def get_run_lists():
+    conn = engine.connect()
+    result = conn.execute("SELECT name, id FROM run_lists ORDER BY name ASC")
+    data = OrderedDict()
+    for entry in result.fetchall():
+        data[str(entry[0])] = int(entry[1])
+    conn.close()
+    return data
+
+def get_timestamp(local_tz=None, local=False, iso=False):
```

Comment:
Is this used anywhere other than in `writeNewLists`? If not please also remove pytz and dateutil from the imports and in `requirements.txt`